arelle/plugin/validate/ESEF/Dimensions.py

```diff
@@ -165,7 +165,7 @@ def addDomMbrs(sourceDomMbr, ELR, membersSet):
                             val.modelXbrl.error("ESEF.3.4.3.extensionTaxonomyOverridesDefaultMembers",
                                 _("The extension taxonomy MUST not modify (prohibit and/or override) default members assigned to dimensions by the ESEF taxonomy."),
                                 modelObject=linkChild)
-                    if modelLink.role != "http://www.esma.europa.eu/xbrl/role/cor/ifrs- dim_role-990000":
+                    if modelLink.role not in DefaultDimensionLinkroles:
```

Comment:
[link](https://github.com/Arelle/Arelle/blob/master/arelle/plugin/validate/ESEF/Const.py#L32) for others ease of review